examples/basic_query/example_client.py

```diff
@@ -24,11 +22,10 @@ def predict(host, uid, x):
 
 if __name__ == '__main__':
     host = "localhost"
-    clipper = cm.Clipper(host, check_for_docker=False)
+    clipper = Clipper(host, check_for_docker=False)
```

Comment:
Because this example doesn't start Clipper, it assumes there is already a Clipper instance running and model connected. We don't check for Docker so that the example works even if you aren't using Docker to run Clipper.